src/main/scala/edu/ie3/osmogrid/lv/region_coordinator/LvRegionCoordinator.scala

```diff
@@ -75,13 +72,16 @@ object LvRegionCoordinator {
                 )
                 .values
 
-          val levels = BoundaryAdminLevel(cfg.boundaryAdminLevel.lowest)
-            .zip(BoundaryAdminLevel.nextLowerLevel(administrativeLevel))
-            .filter((lowestLevel, nextLevel) => nextLevel <= lowestLevel)
+          val levels = BoundaryAdminLevel
+            .get(cfg.boundaryAdminLevel.lowest)
+            .zip(administrativeLevel.nextLowerLevel())
+            .filter(lowestLevelNextLevel =>
```

Comment:
Could also use case `case (lowestLevel, nextLevel) =>` here, as this is more descriptive